src/adapter.ts

```diff
@@ -596,6 +599,43 @@ export class perlDebuggerConnection {
 		return result;
 	}
 
+	async getLoadedFiles(): Promise<string[]> {
+
+		const loadedFiles = await this.getExpressionValue(
+			`join "\t", grep { /^_</ } keys %main::`
```

Comment:
this can be a regular string using ' instead of the interpolated version `

Reply:
Fixed.